src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5506,6 +5506,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_ParameterConditionallyDisallowsNull_Title" xml:space="preserve">
     <value>Parameter may not have a null value when exiting in some condition.</value>
   </data>
+  <data name="WRN_ParameterDisallowsNull" xml:space="preserve">
+    <value>Parameter '{0}' may not have a null value when exiting.</value>
+  </data>
```

Comment:
"Parameter may not have a null value when exiting" could imply the parameter must have a null value but might not. How about "Parameter must be assigned a non-null value before exiting"? Similar comment for `WRN_ParameterConditionallyDisallowsNull`.